tracing-journald/src/lib.rs

```diff
@@ -359,20 +351,11 @@ fn put_metadata(buf: &mut Vec<u8>, meta: &Metadata, span: Option<usize>) {
             },
         );
     }
-    if let Some(n) = span {
-        write!(buf, "S{}_", n).unwrap();
-    }
     put_field_wellformed(buf, "TARGET", meta.target().as_bytes());
     if let Some(file) = meta.file() {
-        if let Some(n) = span {
-            write!(buf, "S{}_", n).unwrap();
-        }
         put_field_wellformed(buf, "CODE_FILE", file.as_bytes());
     }
     if let Some(line) = meta.line() {
-        if let Some(n) = span {
-            write!(buf, "S{}_", n).unwrap();
-        }
         // Text format is safe as a line number can't possibly contain anything funny
         writeln!(buf, "CODE_LINE={}", line).unwrap();
     }
```

Comment:
`CODE_FILE` and `CODE_LINE` are standard fields that may be interpreted by other off-the-shelf tools. How will they behave when there are multiple values for them? If they generally select the a particular value (perhaps the first or last instance?) perhaps we should make sure that's the event's value. If that's not feasible, should we prefix them with `SPAN_` to avoid confusion?

Reply:
Right, it seems more intuitive to record these fields with span prefix. Fixed